styles/theme.css

```diff
@@ -154,9 +155,9 @@ to decide colors
     --docs_spacing_10: 120px;
 
     --docs_font_size_h1: 1.75rem;
-    --docs_font_size_h2: 1.375rem;
-    --docs_font_size_h3: 1.125rem;
-    --docs_font_size_h4: 1rem;
+    --docs_font_size_h2: 1.5rem;
+    --docs_font_size_h3: 1.375rem;
+    --docs_font_size_h4: 1.125rem;
```

Comment:
Can you also review the reading pages on different screen sizes @chrisb-100ms? Also, pages like release notes and API references have a different structure so might need adjustments